homeassistant/components/tuya/climate.py

```diff
@@ -35,14 +36,20 @@
 from .const import DOMAIN, TUYA_DISCOVERY_NEW, DPCode
 
 TUYA_HVAC_TO_HA = {
-    "auto": HVAC_MODE_HEAT_COOL,
+    "auto": HVAC_MODE_AUTO,
```

Comment:
This is incorrect, auto means the temperature isn't settable and uses an AI or schedule.
As far as I am aware, this isn't the case with Tuya.

Reply:
when I select `Auto` mode for the HY368 radiator in the `Tuya Smart` app, then it works according to the schedule.

Reply:
Yes, but that is specific to your device, the Tuya API still allows changing temperature when in auto, which is not according to Home Assistant specifications. Thus this is not a correct change.

Reply:
Yes, but the device will still **automatically** change settings despite manual adjustments. If the mode is automatic why do you artificially want to change it to manual?

Reply:
nie tylko ja zauważam, że `auto` powinno znaczyć `HVAC_MODE_AUTO`, a nie `HVAC_MODE_HEAT_COOL`
https://github.com/tuya/tuya-home-assistant/issues/407#issuecomment-960525651

Reply:
See my last answer above.